code/src/OSFramework/OSStructure/ExtendedDate.ts

```diff
@@ -0,0 +1,7 @@
+// Auxiliary class that helps us define which type of date comes from the platform data (date or datetime)
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSStructure {
+    export class ExtendedDate extends Date {
+        public isDate: boolean;
```

Comment:
I would add a comment here explaining the meaning/purpose of the isDate since it can cause readability issues in a class that extends Date @BenOsodrac @glundgren93

Reply:
@gnbm We have a comment on the top. Are you saying that is not enough? Or you didn't saw it? :)

Reply:
I think it's enough for now